src/OpenTelemetry/Metrics/Exemplar/ExemplarFilters.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="ExemplarFilters.cs" company="OpenTelemetry Authors">
```

Comment:
Do we need this at all? (maybe it would break the existing ownership model)

e.g. https://github.com/open-telemetry/opentelemetry-dotnet/blob/5680f837e277a73d7c9060ada9ed170414181d7a/docs/trace/reporting-exceptions/Program.cs#L32

https://github.com/open-telemetry/opentelemetry-dotnet/blob/5680f837e277a73d7c9060ada9ed170414181d7a/src/OpenTelemetry/Trace/TracerProviderSdk.cs#L390